### PR TITLE
New spec systemctl show qpidd qrouterd

### PR DESCRIPTION
**Pull request by TripleJQK**

### CODE REVIEW

**Comment by eduardocerqueira** [comment]
There is one failure only for rhel-7.5 during systemd scheduler tests. I don't believe it is related to the changes on this PR, so if you need to merge this, go ahead, otherwise I will re-run this test later. We are debugging that test to identify the root cause.

**Comment by eduardocerqueira** [comment]
One test failed "Upload archive failed with status code  503" It seems some issue with Insights API. 

It was reported to slack channel and @kylape should be aware or investigating it. I will wait for few hours and ask to re-run this test.

In meantime if you need to merge this PR the error above should not be related to the changes on this PR so I believe it is safe to proceed.

**Comment by eduardocerqueira** [comment]
test this please

**Comment by bfahr** [comment]
test this please

File: uploader.json [hunk 1]
```diff
@@ -526,6 +526,15 @@
...
         },
+        {
+            "command": "/bin/ps -ef",
+            "pattern": [
+                "CMD",
+                "nginx: master process",
+                "nginx: worker process"
+            ],
+            "symbolic_name": "ps_ef"
+        },
```
Comment (bfahr) [inline uploader.json]: @TripleJQK I believe this is from a different PR.

File: uploader.json.asc [hunk 2]
```diff
@@ -1,17 +1,17 @@
 -----BEGIN PGP SIGNATURE-----
```
Comment (bfahr) [inline uploader.json.asc]: This file should not be in your PR.

File: uploader.v2.json.asc [hunk 3]
```diff
@@ -1,17 +1,17 @@
 -----BEGIN PGP SIGNATURE-----
```
Comment (bfahr) [inline uploader.v2.json.asc]: This file should not be in your PR.

**Comment by eduardocerqueira** [comment]
test this please

**Comment by TripleJQK** [comment]
close as nonrelated commit include

**Pull request closed without merging**